handlers/misttriggers/push_end.go

```diff
@@ -76,6 +79,10 @@ func (d *MistCallbackHandlersCollection) TranscodingPushEnd(w http.ResponseWrite
 		}
 	}
 
+	if err := createDtsh(actualDestination); err != nil {
+		_ = config.Logger.Log("msg", "createDtsh failed", "err", err)
```

Comment:
Could we have more context (e.g IDs) on this log line to help identify what it failed for?

Reply:
Adding `actualDestination` to that log line